### PR TITLE
Explore section with videos and categories

### PR DESCRIPTION
**Pull request by dev-enforced**
In this pull request the reviewer will find the code required to develop categories and videos that will be shown in the explore and home pages of MI-STREAM. Main components where the changes were related to :
1) Video Card component
2) video Menu Options 
3) useOnClickOutside hook
4) CategoryChips co
And the CSS files related to developing these components in a responsive manner.

Here are the live preview links:
https://mi-stream-git-explore-section-with-videos-a-d1ae73-dev-enforced.vercel.app/
https://mi-stream-git-explore-section-with-videos-a-d1ae73-dev-enforced.vercel.app/explore

Category Linking to explore page feature is not present in this PR. It would be added in the next Pr.

### CODE REVIEW

File: src/pages/Homepage/Homepage.jsx [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-import styles from "./Homepage.module.css"
-const arr=[1,2,3];
+import styles from "./Homepage.module.css";
+
 const Homepage = () => {
+  const { categoriesList } = useCategories();
   return (
-    <div>
+    <>
       <section className="image-slider">
```
Comment (ankit9015) [inline src/pages/Homepage/Homepage.jsx:11]: I like that you are using semantic tags

File: src/utilities/VideoDetailGenerators/viewCountFormatter.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const viewCountFormatter = (views) => {
```
Comment (ankit9015) [inline src/utilities/VideoDetailGenerators/viewCountFormatter.js:1]: I like that you are using separate files for helper functions

File: src/components/CategoryChips/CategoryChips.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <div
+      className={`gentle-flex-gap g-flex-align-center ${styles.categories_wrapper}`}
+    >
+      <div className={`pos-relative`}>
+        <input
+          type="radio"
+          name="category-choice"
+          id="All"
+          className={`${styles.category_input}`}
```
Comment (ankit9015) [inline src/components/CategoryChips/CategoryChips.jsx:15]: I like that you are using styles.class approach for className; I learned something new.

**Review by ankit9015 — APPROVED** [review]
Overall the code looks good to merge

**Review by vaishnavipaliya11 — APPROVED** [review]
Liked that you use sematic tags. 
Liked that you exported functions from a different file.
Code looks clean and easy to understand

**Pull request merged**